handlers/common.go

```diff
@@ -44,129 +44,76 @@ func GetValidatorOnlineThresholdSlot() uint64 {
 
 // GetValidatorEarnings will return the earnings (last day, week, month and total) of selected validators
 func GetValidatorEarnings(validators []uint64, currency string) (*types.ValidatorEarnings, map[uint64]*types.Validator, error) {
```

Comment:
**Minor:** This should have an early exit condition:
```
if len(validators) == 0 {
	return nil, nil, errors.New("no validators provided")
}
```

Reply:
makes sense